app-backend/api/src/main/scala/user/Auth0User.scala

```diff
@@ -323,32 +349,142 @@ object Auth0Service extends Config with LazyLogging {
       .flatMap { responseAsAuth0User _ }
   }
 
-  def createAnonymizedUser(
-      userName: String,
+  @JsonCodec
+  final case class ImportResponse(status: String, id: String)
```

Comment:
These case classes are only used in the functions below -- figured it would be better to keep them close

Reply:
I'm pretty ambivalent about `datamodel`'s boundaries at this point, especially for things like this where we're modeling the response from someone else's service that we call in exactly one place.